### ISSUE TITLE
audit-avt1-data-viz-color-palettes: figure element needs label

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1811]
## Environment:

> Operating system: Mac
> Browser: Chrome 
> AVT :one:
> Page: [Data visualization - Color palettes](https://www.carbondesignsystem.com/data-visualization/color-palettes)

## Detailed Description: 

The `<figure>` element does not have an associated label

<img width="1680" alt="Screen Shot 2020-09-22 at 3 44 42 PM" src="https://user-images.githubusercontent.com/32720851/93935729-b48e6880-fcea-11ea-90e6-1fc2b7b62e48.png">

## Why is this important? 
A <figure> element is usually used to annotate illustrations, diagrams, photos, code listings, etc. Associating a label makes the same information available through audio or other channels.

## What to do: 
Add a <figcaption> element inside the <figure> element to provide a visual caption that describes the figure;
